src/main.cpp

```diff
@@ -63,6 +64,11 @@ sc_int main(sc_int argc, sc_char * argv[])
 
   try
   {
+    if (!options.Has({"interactive", "i"}))
+    {
+      std::cout << "Shutting down, not interacting mode\n";
```

Comment:
```suggestion
      SC_LOG_INFO( "Shutting down, not interacting mode");
```
Add // TODO: implement interactive mode